utils/scarb-ui/src/args/verbosity.rs

```diff
@@ -63,7 +74,12 @@ impl VerbositySpec {
     }
 
     fn integer_verbosity(&self) -> i8 {
-        let int_level = (self.verbose as i8) - (self.quiet as i8);
+        let int_level = if self.no_warnings {
+            -1
+        } else {
+            (self.verbose as i8) - (self.quiet as i8 * 2)
```

Comment:
Let's just conditionally subtract 1 if quiet is used instead of multiplication

Reply:
Do you mean something like:

```suggestion
            (self.verbose as i8) - (self.quiet as i8) - (if self.quiet > 0 { 1 } else { 0 })
```

Not sure if it's better, but maybe we can just do:
```suggestion
            self.verbose as i8) - (self.quiet as i8) - (self.quiet as i8)
```
if you specifically don't like multiplication

Reply:
I just want this level to represent the number of times `-q` has been selected, starting from `-2`, so it's `-q = -2`, `-qq = -3`, `-qqq = -4` etc. The values we get now do not represent the actual integer level we mean by the input.